app/buck2_forkserver/src/run/mod.rs

```diff
@@ -337,11 +395,19 @@ impl KillProcess for DefaultKillProcess {
                 return kill_process_impl(pid, self.graceful_shutdown_timeout_s).await;
             }
         }
-        // `start_kill` is just `std::process::Child::kill` on Windows.
-        // Ignore the error because `kill` fails on Windows if the process has been terminated
-        // even if we did not wait for it.
-        let _ignore = child.start_kill();
-        Ok(())
+
+        #[cfg(windows)]
+        {
+            if true {
```

Comment:
Why `if true` ?

Reply:
I just copied the above code for Unix. I have no idea if I should add an `if true`  or not. The unix code is here: https://github.com/facebook/buck2/blob/22b925eec6e4da59a2d9c6ca202aa2c126c258fa/app/buck2_forkserver/src/run/mod.rs#L331